draft-ietf-mls-architecture.md

```diff
@@ -1608,8 +1616,8 @@ incorrect or attacker-provided identities to clients.
 
 - The attacker can publish or distribute credentials
 
-In the past, some systems have had a centralized server generate signature key
-pairs and distribute them to clients.  In such cases, the centralized server is
+It is possible for a system to have a centralized server generate signature key
```

Comment:
```suggestion
It is possible for a system to have a central server generate signature key
```

Reply:
Keeping centralized for consistency (appears twice more in the same paragraph)